be/src/vec/data_types/serde/data_type_ipv6_serde.cpp

```diff
@@ -69,25 +69,17 @@ Status DataTypeIPv6SerDe::write_column_to_mysql(const IColumn& column,
 }
 
 void DataTypeIPv6SerDe::read_one_cell_from_jsonb(IColumn& column, const JsonbValue* arg) const {
-    IPv6 val = 0;
-    const auto* str_value = static_cast<const JsonbStringVal*>(arg);
-    ReadBuffer rb(reinterpret_cast<const unsigned char*>(str_value->getBlob()),
-                  str_value->getBlobLen());
-    if (!read_ipv6_text_impl(val, rb)) {
-        throw doris::Exception(ErrorCode::INVALID_ARGUMENT, "parse ipv6 fail, string: '{}'",
-                               rb.to_string());
-    }
-    assert_cast<ColumnIPv6&>(column).insert_value(val);
+    const auto ip_value = static_cast<const JsonbInt128Val*>(arg)->val();
+    const IPv6* ip_val = reinterpret_cast<const IPv6*>(&ip_value);
+    assert_cast<ColumnIPv6&>(column).insert_value(*ip_val);
 }
 
 void DataTypeIPv6SerDe::write_one_cell_to_jsonb(const IColumn& column,
                                                 JsonbWriterT<JsonbOutStream>& result,
                                                 Arena* mem_pool, int col_id, int row_num) const {
-    // we make ipv6 as string in jsonb
     result.writeKey(col_id);
     IPv6 data = assert_cast<const ColumnIPv6&>(column).get_element(row_num);
-    IPv6Value ipv6_value(data);
-    result.writeString(ipv6_value.to_string());
+    result.writeInt128(int128_t(data));
```

Comment:
using IPv6 = uint128_t;
If it is an uint128, why it could be convert to int128_t??